source/physics/include/TG4TrackInformation.h

```diff
@@ -69,6 +76,13 @@ class TG4TrackInformation : public G4VUserTrackInformation
     G4double fPDGEncoding;     ///< the particle PDG encoding
     G4bool   fIsUserTrack;     ///< true if defined by user and not primary track
     G4bool   fStop;            ///< true if track should be stopped
+    G4bool   fInterrupt;       ///< true if track should be interrupted
+    TMCParticleStatus* fInitialTrackStatus; ///< Pointer holds some additional
+                                               ///< information on the particle
```

Comment:
For long comments, prefer the style with the comment above the data meber:
/// Comment
TMCParticleStatus* fInitialTrackStatus;